source/common/filter/http/filter_config_discovery_impl.cc

```diff
@@ -141,6 +141,9 @@ void FilterConfigSubscription::onConfigUpdate(
       }
     });
   }
+  last_config_ = filter_config;
```

Comment:
I wonder if we need to take a special care of large configs (like Wasm inlined bytecode).

Reply:
This will need to be handled case by case I suppose? I can add special case logic for Wasm filter config into the config dump function, to redact the inline bytes. wdyt?

Reply:
Maybe some runtime override? I don't like hardwriting Wasm specifically. I think we might get quick a bump in memory use if Wasm is shipped via ECDS (it's not atypical to see >10MB per module).